pages/data.css

```diff
@@ -0,0 +1,20 @@
+.data-tab {
+  width: 100%;
+}
+
+.data-tab tr {
+  cursor: pointer;
+}
+
+.label-small {
+  max-width: 5rem;
+  overflow: hidden;
+  white-space: nowrap;
+  transition: width 1.5s;
+}
+
+.label-expanded {
+  max-width: 100%;
+  overflow: initial;
+  transition: max-width 1.5s;
+}
```

Comment:
The idea is to parse OAI and display only ID part hiding others. When it gets expanded, other parts become visible.

In terms of HTML it's something like:
```html
<span class="label primary">
  <span class="oai-part-generic">
    <span class="oai-part-protocol">oai</span>
    <span class="oai-part-domain">oro.open.ac.uk</span>
  </span>
  <span class="oai-part-id">1</span>
</span>